packages/frontend/src/ee/components/Home/AiSearchBox/SearchDropdown.tsx

```diff
@@ -115,16 +125,22 @@ export const SearchDropdown: FC<Props> = ({
                         scrollbars="y"
                         className={styles.searchDropdownScrollContent}
                     >
-                        {!isSuccess ? (
+                        {isFetching ? (
                             <Combobox.Empty>
-                                <Group align="center" justify="center" gap="xs">
-                                    <Loader size={12} color="gray" />
-                                    Loading results
+                                <Group
+                                    align="center"
+                                    justify="flex-start"
+                                    gap="xs"
+                                >
+                                    <Loader size={12} />
+                                    Loading spaces/dashboards/charts
```

Comment:
maybe this should say "Loading search results"?

Reply:
I'm following the figma file - this copy was requested by @PriPatel

Reply:
The copy in the file was[ following the spec](https://linear.app/lightdash/project/ai-homepage-visibility-bdb3f917c42d/overview) where we decided that only spaces/dashboards/charts should be shown. However, all search results were implemented, which i dont think is a problem, but yes the copy now has to be generic.